packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -289,26 +326,32 @@ export function renderCurrent ({ current, wanted }: OutdatedPackage) {
   return `${output} (wanted ${wanted})`
 }
 
-export function renderLatest (outdatedPkg: OutdatedWithVersionDiff): string {
+export function renderLatest (outdatedPkg: OutdatedWithVersionDiff, withoutChalk = false): string {
```

Comment:
I don't see any benefit from using this function. Just create a new one.

Reply:
I was concerned about a scattering of implementation, but this is not a firm intention, so fixed in [90069a4](https://github.com/pnpm/pnpm/pull/5582/commits/90069a4fa1545f84df758f765869f02c31eed3e2) 👍